storage/innobase/handler/ha_innodb.cc

```diff
@@ -9490,13 +9385,14 @@ ha_innobase::write_row(
 
 		m_num_write_row = 0;
 
-		switch (
-		    wsrep_run_wsrep_commit(m_user_thd, wsrep_hton, true)) {
+		int ret=0;
```

Comment:
Wouldn't the following be more readable?
```
int ret = foo();
switch(ret) { ...
```

Reply:
Agreed. Done.